README.md

```diff
@@ -1,2 +1,28 @@
-# discuss
-Unikraft community discussions
+# Unikraft Meetings
+
+This is a repository to store all of the past and future Unikraft community discussions. 
+
+To contribute to this repository, please follow these guidelines:
+ * Each meeting will follow the `markdown` format.
+ * Each meeting entry will have the name format `yyyy-mm-dd.md` and will be placed in the corresponding meeting and year subfolders.
+ * If a meeting is archived, the name of the subfolder should be changed to `name-archived`.
+ * If a discussion is not recurring, place it in the `other` subfolder with a suggestive name: `name-yyyy-mm-dd.md`.
+ * Only include public meeting discussions.
+
+Each meeting file will follow the structure below.
+When creating a new meeting entry, copy this structure and fill in the contents replacing `TO FILL`.
+
+```md
+## Date, Time:
+TO FILL
+## Location:
+TO FILL
```

Comment:
Hi,

I'd suggest turning these two into a markdown meta list, e.g.:

```
---
datetime: 2021-01-01 12:00:00
location: Discord, `#monkey-business`
---
```

This will be rendered as a small table at the top of any markdown table.

You can see a random example render [here](https://github.com/gohugoio/hugo/blob/master/docs/content/en/_index.md).

By doing this, we can later easily parse these pieces of information, say, for example, if they were to be included on the website / docs site :)

Reply:
We could also add relevant teams or participant aliases, e.g.:
```
---
datetime: 2021-01-01 12:00:00
location: Discord, `#monkey-business`
teams:
  - sig-arch-x86
participants:
  - craciunoiuc
  - razvand
  - nderjung
---
```
Long lists, do, however, have the caveat of being rendered as horizontal tables in the markdown format though.

Reply:
This table looks really interesting. I'm okay with it, also considering your parsing argument.

Let's see what @razvand says

Reply:
Agree with @nderjung , let's do that.